package.json

```diff
@@ -46,36 +46,36 @@
     "@babel/preset-typescript": "7.16.7",
     "@mui/material": "^5.9.1",
     "@mui/styled-engine-sc": "^5.8.0",
-    "@mui/styles": "^5.9.1",
-    "@nrwl/cli": "15.0.0",
-    "@nrwl/cypress": "15.0.0",
-    "@nrwl/eslint-plugin-nx": "15.0.0",
-    "@nrwl/express": "15.0.0",
-    "@nrwl/jest": "15.0.0",
-    "@nrwl/linter": "15.0.0",
-    "@nrwl/node": "15.0.0",
-    "@nrwl/react": "15.0.0",
-    "@nrwl/tao": "15.0.0",
-    "@nrwl/web": "15.0.0",
-    "@nrwl/workspace": "15.0.0",
+    "@nrwl/js": "16.0.3",
+    "@nrwl/tao": "16.0.3",
+    "@nx/cypress": "16.0.3",
+    "@nx/eslint-plugin": "16.0.3",
+    "@nx/express": "16.0.3",
+    "@nx/jest": "16.0.3",
+    "@nx/linter": "16.0.3",
+    "@nx/node": "16.0.3",
+    "@nx/web": "16.0.3",
```

Comment:
Looks like you don't need `@nx/web` package, which you actually stopped using when you upgraded to Nx 15 in 9f2cb06fb85e935208b60b75b7da3f1abbc44145